app/controllers/concerns/catalog.rb

```diff
@@ -35,6 +36,28 @@ def search_results(user_params, search_params_logic)
     [response, documents]
   end
 
+  # Overrides {Blacklight::SearchHelper} method to detect Collections searches
+  def get_previous_and_next_documents_for_search(index, request_params, extra_controller_params = {})
```

Comment:
Assignment Branch Condition size for get_previous_and_next_documents_for_search is too high. [24.23/15]<br>Method has too many lines. [13/10]